src/components/AppLayout/AppHeader.tsx

```diff
@@ -994,61 +1177,119 @@ export function AppHeader({
                       </ScrollArea.Autosize>
                     ) : (
                       <>
-                        {/* Calculate maxHeight based off total viewport height minus header + footer + static menu options inside dropdown sizes */}
-                        <ScrollArea.Autosize maxHeight={'calc(100dvh - 135px)'}>
+                        <Group
+                          sx={{ display: 'flex', alignItems: 'center', padding: '10px', justifyContent: 'space-between' }}
```

Comment:
I'm just curious, you started using tailwind classes for styling then kept using mantine 😛 there's nothing wrong with it, but it'b be good to keep it consistent. If you think it'll be a pain in the ass, just omit this comment. This is just nitpicking